src/modules/clients/ClientsList.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect } from 'react';
+import { getClientByBusinessId } from '../inventory/features/Clients/clientSlice';
+import { useAppDispatch, useAppSelector } from '../../Redux/hooks';
+import { styled } from '@mui/material/styles';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell, { tableCellClasses } from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+const StyledTableCell = styled(TableCell)(({ theme }) => ({
+  [`&.${tableCellClasses.head}`]: {
+    backgroundColor: theme.palette.common.black,
+    color: theme.palette.common.white,
+  },
+  [`&.${tableCellClasses.body}`]: {
+    fontSize: 14,
+  },
+}));
+
+const StyledTableRow = styled(TableRow)(({ theme }) => ({
+  '&:nth-of-type(odd)': {
+    backgroundColor: theme.palette.action.hover,
+  },
+  '&:last-child td, &:last-child th': {
+    border: 0,
+  },
+}));
+
+function ClientsListByBusinessId() {
+  const dispatch = useAppDispatch();
+  const clientsData = useAppSelector((state) => state.client.data);
+  const currentUser = useAppSelector((state) => state.currentUserSlice);
+  const businessId = currentUser?.employeeDetails.businessId?.toString();
+  useEffect(() => {
+    //You need to check that the businessID does return
+    if (businessId) {
+      dispatch(getClientByBusinessId(businessId));
+    }
+  }, [dispatch]);
+
+  return (
+    <div>
+      <TableContainer component={Paper}>
+        <Table sx={{ minWidth: 700 }} aria-label='customized table'>
+          <TableHead>
+            <TableRow>
+              <StyledTableCell>Username</StyledTableCell>
+              <StyledTableCell>Email</StyledTableCell>
+              <StyledTableCell>Registered At</StyledTableCell>
+              <StyledTableCell>Last Login</StyledTableCell>
```

Comment:
בבקשה תשתמשי ברכיב טבלה שיש בסטוריבוק

Reply:
בנוסף, נא לתרגם את הטקטסים

Reply:
נחמי אמרה לי לקחת משהו מmui
ומה הכוונה לתרגם את הטקסטים?

Reply:
יש רכיב יחסית חדש של טבלה בסטוריבוק
בבקשה תשתמשי בו

Reply:
כמה מאמץ יהיה לשנות את זה?

Reply:
הרבה ניסיתי אני לא יודעת מה הולך שם זה כזה דחןף שאני יבזבז על זה זמן?

Reply:
אני חושבת שהיום חבל על ההשקעה בזה
@buzz-code מסכימה איתי?

Reply:
@nechamaLearn 
אני מסכימה שחבל להשקיע בזה הרבה זמן
אבל לפי איך שאני זוכרת את הקומפוננטה, להשתמש בה יהיה סיפור של 20-30 דקות ולא יותר

@racheliNusboum 
האם ניסית לפנות לבת שפיתחה את הרכיב כדי שתעזור לך להשתמש בו?
בכל מקרה, אם את רואה שזה לוקח לך יותר מדי זמן, תוסיפי הערה של TODO: move to use table component from storybook ונעלה ככה